packages/web/src/app/graph/page.tsx

```
@@ -1,30 +1,129 @@
+    ? makeKeys(selectedNode.doi, selectedNode.title).some((k) =>
+        savedKeys.has(k),
+      )
+    : false;
+
+  const markSelectedSaved = useCallback(() => {
+    if (!selectedNode) return;
+    const keys = makeKeys(selectedNode.doi, selectedNode.title);
+    if (keys.length === 0) return;
+    setSavedKeys((prev) => {
+      const next = new Set(prev);
+      keys.forEach((k) => next.add(k));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)